examples/nodejs/storage/basic.ts

```diff
@@ -1,17 +1,12 @@
 import {
   CreateStoreResponse,
-  CredentialProvider,
   PreviewStorageClient,
-  StorageConfigurations,
   StorageGetResponse,
   StoragePutResponse,
 } from '@gomomento/sdk';
 
 async function main() {
-  const storageClient = new PreviewStorageClient({
-    configuration: StorageConfigurations.Laptop.latest(),
-    credentialProvider: CredentialProvider.fromEnvironmentVariable('MOMENTO_API_KEY'),
-  });
+  const storageClient = new PreviewStorageClient({});
```

Comment:
should the props object itself become optional then? 
removing the `{}` results in an `Expected 1 arguments, but got 0` error.

Reply:
good call, we could do that later, but I don't think it's a blocker for now.